### PR TITLE
Security

### PR DESCRIPTION
**Pull request by evilnick**
The secuirty overview still needs re-writing but I think this is the 'minimum viable pull request' which satisfies that security card

### CODE REVIEW

File: _data/sidebars/k8smain-sidebar.yml [hunk 1]
```diff
@@ -289,6 +294,11 @@ entries:
       output: web, pdf
       type: page
 
+    - title: Using the OPA Gatekeeper charm
```
Comment (evilnick) [inline _data/sidebars/k8smain-sidebar.yml:297]: Note the comment on the card - Not sure whether people will be looking for 'gatekeeper' or 'policy management'

Comment (kwmonroe) [inline _data/sidebars/k8smain-sidebar.yml:297]:
```suggestion
    - title: OPA Gatekeeper
```
I think we should shorten this.  The phrase "using the ... charm" doesn't bring much to the table imho.  This is also a bit more consistent with some of our other headings like `[keepalived|HAcluster|MetalLB]`.

Comment (kwmonroe) [reply _data/sidebars/k8smain-sidebar.yml:297]: For now, i think gatekeeper is appropriate.  We can cross the more generic "policy mgmt" bridge if we ever support something else.

File: pages/k8s/security.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ toc: False
 ---
 
 This page provides an overview of various aspects of security to be considered
-when operation a **Charmed Kubernetes** cluster. To consider security properly,
+when operating a **Charmed Kubernetes** cluster. To consider security properly,
```
Comment (kwmonroe) [inline pages/k8s/security.md:17]: :heart:

**Review by kwmonroe — APPROVED** [review]
Minor suggestion to the sidebar title, but otherwise LGTM. Thanks!

File: _data/sidebars/k8smain-sidebar.yml (hunk 1 above)
Comment (kwmonroe) [reply _data/sidebars/k8smain-sidebar.yml:297]: I think gatekeeper is fine for now.  We can cross the "policy mgmt" bridge if/when we support something besides opa.

**Pull request merged**